src/lib/components/auth-cards/CreateCard.svelte

```diff
@@ -107,21 +116,27 @@
 
   const setMessage = () => {
     let emailAlreadyExist = "auth/email-already-in-use";
+    let invalidEmail = "auth/invalid-email"
     let isExistingEmail;
+    let isInvalidEmail
 
     isExistingEmail = fireBaseErr.indexOf(emailAlreadyExist);
     if (isExistingEmail > -1) {
       createErrText = "Account already exist. Please sign in.";
     }
 
+    isInvalidEmail = fireBaseErr.indexOf(invalidEmail);
+    if (isInvalidEmail > -1) {
+      invalidEmailText = "Email is invalid. Please enter a valid email.";
+    }
+
     localStorage.removeItem("createErr");
     setTimeout(() => {
       resetState();
-    }, 10000);
+    }, 15000);
```

Comment:
Why do we need a `setTimeout` here btw? I see that it's not new but just curious. Is the intent to not block synchronously on accessing local storage?

Reply:
Oh no, I did that so that error message behaves like a temporary one. Once the time elapses, it disappears. I have no preference if we remove it and the messages stay until the user does an appropriate action. And actually I added a change to simplify it all.